lib/modules/platform/types.ts

```diff
@@ -242,7 +242,14 @@ export interface Platform {
   ensureIssue(
     issueConfig: EnsureIssueConfig,
   ): Promise<EnsureIssueResult | null>;
-  massageMarkdown(prBody: string): string;
+  massageMarkdown(
+    prBody: string,
+    /**
+     * Useful for suggesting the use of rebase label when there is no better
+     * way, e.g. for Gerrit.
+     */
+    rebaseLabel?: string,
```

Comment:
I've thought about it some more, and I'd like to refactor the markdown text so that it mentions the rebase label for all platforms

Reply:
I've split this off into a separate discussion: https://github.com/renovatebot/renovate/discussions/36130

Reply:
let's remove this refacto from this pr?

Reply:
Yes